RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,7 @@
+# Release procedure
+
+- Update [version_history.md](docs/root/version_history.md). 
```

Comment:
(optional) Not sure if worth it (depends on how often this happens). Would it make sense to add a script that will automate the version update and the version "Bump" noted below?